.github/.staging_workflows/anchored_sciops_docker_images.yaml

```
@@ -76,31 +73,41 @@ anchor-docker-login: &docker-login
 
 jobs:
   ## Build/Publish codebook env image
-  build_env_image:
+  build_codebook_image:
     runs-on: ubuntu-latest
     env:
       JHUB_VER: ${{inputs.jhub_ver}}
       PY_VER: ${{inputs.py_ver}}
       DIST: ${{inputs.dist}}
-      ENV_BASE_HASH: ${{inputs.env_base_hash}}
+      CODEBOOK_BASE_HASH: ${{inputs.codebook_base_hash}}
       WORKFLOW_VERSION: ${{inputs.workflow_version}}
     steps:
-      - uses: actions/checkout@v2
-      - id: compile_env
-        name: Compile codebook env image
+      - name: Checkout workflow repo
+        uses: actions/checkout@v3
+        with:
+          path: 'main'
+      - name: Checkout cicd repo that has common docker template
+        uses: actions/checkout@v3
+        with:
+          repository: 'dj-sciops/djsciops-cicd'
+          path: 'cicd'
+      - id: build_env
+        name: Build codebook env image
         run: |
+          ## Get docker template
+          cp -r ./cicd/docker-template/* ./main/docker/
+
           ## Get build info
-          cd ./docker
-          export REPO_NAME=$(echo ${{github.repository}} | awk -F\/ '{print $2}')
-          echo "::set-output name=repo_name::${REPO_NAME}"
+          cd ./main/docker/codebook_env
+          export REPO_NAME=${{github.event.repository.name}}
           export REPO_OWNER=${{github.repository_owner}}
 
           echo "${{secrets.RAW_DEPLOY_KEY}}" | base64 -di >> ${REPO_NAME}-deploy.pem
           chmod 400 ${REPO_NAME}-deploy.pem
```

A bit more verbose but think this is a bit easier for reviews.

```suggestion
          chmod u=r,g-rwx,o-rwx ${REPO_NAME}-deploy.pem
```